entregas/baquedanoBryan/reto-001/Reto01.java

```diff
@@ -0,0 +1,39 @@
+public class    Reto01{
+    public static void main(String[] args) {
+        int precio = 169;
+        int pago = 200;  
+
+       
+        int cambio = pago - precio;
```

Comment:
Espacios innecesarios
```suggestion
        int pago = 200;  
        int cambio = pago - precio;
```